subset/switches/src/main/java/switchtest/Main.java

```diff
@@ -1,50 +1,46 @@
 package switchtest;
 
-import switchtest.allied.AlliedTelesisX230;
-import switchtest.cisco.Cisco9300;
+import grpc.SwitchInfo;
+import grpc.SwitchModel;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
 
 public class Main {
 
-  public static void main(String[] args) throws Exception {
+  /**
+   * Switch test runner.
+   * @param args args
+   */
+  public static void main(String[] args) {
 
-    if (args.length < 4) {
+    if (args.length < 5) {
       throw new IllegalArgumentException(
-          "Expected ipAddress && port && supportPOE && switchModel as arguments");
+          "usiUrl, rpcTimeoutSec switchIpAddress, port, supportPOE, switchModel [username] [password] as arguments");
```

Comment:
something about this string is not right. Lost "expected"?  Some things have commas others don't?